javascript/javascript1/week1/pizza-exercise.js

```diff
@@ -0,0 +1,16 @@
+console.log("I love pizza");
+let pizzaName = "chicken-pizza";
+let pizzaPrice =10;
```

Comment:
Be consistent with the code style. The variables on the other lines have a space between the assingment (`=`) and the value.